lib/Graph/Grad.cpp

```diff
@@ -30,6 +31,9 @@ using namespace glow;
 using llvm::cast;
 using llvm::isa;
 
+#define DECORATE_NODE_NAME(Node, ...)                                          \
```

Comment:
Nit: Why is this macro defined in two places? Should this definition moved into one place, e.g. into a header? Also, does it need to be a macro? It seems like LLVM defines join_items as a template function with variadic arguments.

Reply:
It has slightly different semantics in both places (ptr vs ref). Also figured it was maybe better not to pollute the glow namespace with this helper.

Macro instead of just calling llvm::join_items to avoid replicating the `Node->getName()` everywhere.